CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ _None_
   [David Jennes](https://github.com/djbe)
   [#562](https://github.com/SwiftGen/SwiftGen/issues/562)
   [#797](https://github.com/SwiftGen/SwiftGen/pull/797)
+* Strings: objc-h now emits documentation comments  
+  [@szotp](https://github.com/szotp) 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/72a38125677fd95e2a4fc395ab42994cb4df2a1b/CHANGELOG.md#L24"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>

Reply:
fixed